.github/workflows/lint.yml

```diff
@@ -1,13 +1,14 @@
 name: Rubocop
```

Comment:
I wanted to rename this workflow to `Lint` to match [slack-api-ref](https://github.com/slack-ruby/slack-api-ref/blob/master/.github/workflows/lint.yml) and so we could add other linters to this workflow if needed, but GitHub wouldn't recognize the history of previous runs. Maybe we can rename this in the future.

Reply:
No strong opinions.